src/sdk/base/src/main/java/owt/base/PeerConnectionChannel.java

```diff
@@ -466,6 +468,18 @@ public void onCreateSuccess(final SessionDescription sessionDescription) {
     @Override
     abstract public void onSignalingChange(PeerConnection.SignalingState signalingState);
 
+    @Override
+    public void onStandardizedIceConnectionChange(PeerConnection.IceConnectionState newState) {
```

Comment:
It seems to be an API change although it's not a public API. Is there any class inherited from this one cannot implement these methods?

Reply:
@taste1981, please take a look at these methods as well. If it doesn't work for testing app, it may not work for sample app as well?

Reply:
the default impl in the interface is in org.webrtc package, so it's not visible in owt.base package or owt.p2p pckage. override the default one is neccessary.

Reply:
Thanks for your change. We confirmed this issue. These methods are called by native code which is not in the same package, so it should be public here.